game/scripts/npc/abilities/ember_spirit_sleight_of_fist.txt

```diff
@@ -38,7 +38,7 @@
       "02"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "bonus_hero_damage"                               "40 80 120 160 270 550"
+        "bonus_hero_damage"                               "40 80 120 160 480 800"
```

Comment:
This bonus damage at OAA levels is kinda ridiculous for a basic spell. Can you tone it down to 320/640? He has 3 sleight of fist charges as well.

Reply:
Toned it back. He only has 2 charges on sleight though.

Reply:
Oh yeah, my bad. It always felt like he has 3.